README.md

```diff
@@ -117,6 +117,31 @@ Follow the given instructions for Login into the app.
 
 ## Environment Variables
 
+Make sure you create .env using .env.template and update the values of corresponding environment variables:
+
+```
+SECRET_KEY=<your-secret-key>
+EMAIL_USER=<email-address>
+EMAIL_PASSWORD=<your-password>
+
+```
+
+This environment variable is optional for development
+```
+SENDGRID_API_KEY=<your-sendgrid-api-key>
+```
+
+Where the database is necessary, you will need to provide the DB details in the .env file
+
+ ```
+DB_BACKEND=postgresql
+DB_NAME=<db-name>
+DB_USERNAME=<db-user-username>
+DB_PASSWORD=<db-user-password>
+DB_HOST=localhost
+DB_PORT=5432
+```
+
```

Comment:
We don't need that much info for the template file. Just add info about there is a template file that you can use for updating or to use env variables.
Please change the context of the above sentence to something meaningful.

Reply:
> Please change the context of the above sentence to something meaningful.

@codesankalp here you're talking about changing the database context or is it also for the template?

Reply:
I was saying to simplify it and finish it in 2 to 3 lines in README as there is a section for env variables already in readme we don't need that much big detail for templates.